tmate-slave.c

```diff
@@ -188,6 +192,9 @@ int main(int argc, char **argv, char **envp)
 
 	setup_locale();
 
+	if (!tmate_settings->bind_ip)
+		tmate_settings->bind_ip = "0.0.0.0";
```

Comment:
and if we wanted to listen on all interfaces (ipv4, ipv6), would the `0.0.0.0` work?
I'd think not. Maybe it's better to leave it NULL ?